webpack/envs/baseConfig.js

```diff
@@ -92,6 +93,9 @@ exports.baseConfig = {
       waypoints: "jquery-waypoints/waypoints.js",
     }),
     new LoadablePlugin(),
+    new RetryChunkLoadPlugin({
+      maxRetries: 5,
```

Comment:
```suggestion
      maxRetries: 5,
      cacheBust: `function() {
        return "cache-bust=" + Date.now();
      }`,
```

I pulled this straight from their docs. When retrying, it'll append a timestamp instead of the default `?cache-bust=true` it'll append something like `?cache-bust=1583600489608` which has less chance of secondary failures.

Reply:
It kinda feels like this should be a part of webpack's core /shrug